shared/api/image.go

```diff
@@ -41,6 +41,9 @@ type ImagePut struct {
 	AutoUpdate bool              `json:"auto_update" yaml:"auto_update"`
 	Properties map[string]string `json:"properties" yaml:"properties"`
 	Public     bool              `json:"public" yaml:"public"`
+
+    // API extension: images_expiry
+	ExpiresAt  time.Time         `json:"expires_at" yaml:"expires_at"`
```

Comment:
The comment should be using a tab rather than spaces for indent.
The ExpiresAt line also doesn't match Go formatting standard.

Both of those issues can be resolved by running `go fmt ./...`.

Reply:
I resolved this issue, I have since changed my vimrc to not auto-insert spaces. Do you want us to manicure the git commit history to exclude the additional commits or are we just going to squash them in the end anyways?

Reply:
You're going to need to rebase the branch, if only to edit the `Fixed formatting` commit message as that one is missing the `Signed-off-by` line.

I'd recommend merging the two follow-up commits into the original shared/api commit.

Thanks!